linode/manager
# Release v1.113.0 - `release` → `staging`

## Cloud Manager
### [2024-02-20] - v1.113.0

### Added:

- Feb 2024 Marketplace apps ([#10149](https://github.com/linode/manager/pull/10149))

### Changed:

- Improve Linode Graph X Axis Labels when viewing historic data ([#10186](https://github.com/linode/manager/pull/10186))

### Fixed:

- EditableText interaction styling ([#10132](https://github.com/linode/manager/pull/10132))
- Inability to transfer IPv6 ranges ([#10156](https://github.com/linode/manager/pull/10156))
- Incorrect `X-Filter` on the Account Maintenance "Pending" Table (#10196)
- Bundle analyzer script ([#10175](https://github.com/linode/manager/pull/10175))

### Tech Stories:

- Clean up DC Get Well feature flag logic ([#10146](https://github.com/linode/manager/pull/10146))
- Clean up `regionDropdown` feature flag ([#10148](https://github.com/linode/manager/pull/10148))
- Update `react-router-dom` in preparation for React 18 ([#10154](https://github.com/linode/manager/pull/10154))
- Remove Enzyme ([#10160](https://github.com/linode/manager/pull/10160))
- Update Luxon ([#10163](https://github.com/linode/manager/pull/10163))
- Update `launchdarkly-react-client-sdk` ([#10165](https://github.com/linode/manager/pull/10165))
- Add analytics event for breadcrumb label edit icon on Linode details page ([#10183](https://github.com/linode/manager/pull/10183))

### Tests:

- Add integration test coverage for Account Login History ([#10125](https://github.com/linode/manager/pull/10125))
- Add integration test to check proxy user disabled username/email field ([#10139](https://github.com/linode/manager/pull/10139))
- Add Cypress tests for OBJ Multicluster access key operations ([#10144](https://github.com/linode/manager/pull/10144))
- Fix billing contact Cypress test by narrowing element selection scope ([#10150](https://github.com/linode/manager/pull/10150))
- Update Cypress tests to use `"default"` `user_type` for non-parent/child/proxy users ([#10176](https://github.com/linode/manager/pull/10176))
- Fix Button enabled assertions ([#10142](https://github.com/linode/manager/pull/10142))

### Upcoming Features:

- Disable "Save" button in Edit Access Key drawer unless field values are changed ([#10118](https://github.com/linode/manager/pull/10118))
- Add Placement Groups Select component (#10100)
- Update Placement Groups limits ([#10191](https://github.com/linode/manager/pull/10191))
- Add Placement Group Linodes List ([#10123](https://github.com/linode/manager/pull/10123))
- Add AssignLinodesToPlacementGroup drawer ([#10140](https://github.com/linode/manager/pull/10140))
- Add PlacementGroups Summary component ([#10164](https://github.com/linode/manager/pull/10164))
- Add unassign linode from Placement Group modal (#10172)
- Improve restricted access Login History experience for child and restricted users ([#10125](https://github.com/linode/manager/pull/10125))
- Add session expiry confirmation dialog for proxy to parent user account switching ([#10152](https://github.com/linode/manager/pull/10152))
- Clean up files to use profile to get `user_type` ([#10102](https://github.com/linode/manager/pull/10102))
- Update components and unit tests to use `"default"` `user_type` for non-parent/child/proxy users ([#10176](https://github.com/linode/manager/pull/10176))
- Use infinite query for fetching child accounts ([#10179](https://github.com/linode/manager/pull/10179))
- Use API filtering on user_type to populate the two Users & Grants tables ([#10192](https://github.com/linode/manager/pull/10192))

## APIv4
### [2024-02-20] - v0.110.0

### Upcoming Features:

- Update /account and /profile UserType from `null` to `"default"` ([#10176](https://github.com/linode/manager/pull/10176))
